scan_to_inventory/static/www/controllers/c_load.js

```diff
@@ -0,0 +1,55 @@
+angular.module('scan_to_inventory').controller(
+        'LoadCtrl', [
+        '$scope', '$rootScope', 'jsonRpc', '$state', 'ProductProductModel', 'StockLocationModel', 'StockInventoryModel', '$translate',
+        function ($scope, $rootScope, jsonRpc, $state, ProductProductModel, StockLocationModel, StockInventoryModel, $translate) {
+
+    $scope.data = {};
+
+    $scope.$on(
+            '$stateChangeSuccess',
+            function(event, toState, toParams, fromState, fromParams){
+        if ($state.current.name === 'load') {
+            $scope.data.product_ok = false;
+            $scope.data.product_ko = false;
```

Comment:
1. product_ok and product_ko are both used in v_load.html. (and use ng-hide / ng-show)
There are 3 state in fact : 
- product_ok false / product_ko false : loading 
- product_ok True / product_ko false : OK
- product_ok False / product_ko True : KO

Is it OK, if I just set a state $scope.data.product_state : 'loading' / 'ko' / 'ok' , and set ng-show="product_state == 'ok'", in load view ?